stl/inc/yvals_core.h

```diff
@@ -1409,6 +1410,7 @@
 #define __cpp_lib_latch                   201907L
 #define __cpp_lib_list_remove_return_type 201806L
 #define __cpp_lib_math_constants          201907L
+#define __cpp_lib_move_iterator_concept   202207L

```

Comment:
Not sure whether to change. Is P2520R0 "Yet another DR against C++20"?

Reply:
Nicole and I discussed this in person. This paper is effectively an admission that artificially depressing the category of `move_iterator` was a failed experiment. There's no value in preserving the undesired behavior in C++20 mode, and we suspect the potential for breakage to be very small given that `move_iterator`s are still input iterators even if they now sometimes satisfy stronger concepts. We therefore decided to implement this change in all pertinent language modes.

Reply:
😸Thank you for explanation! It sounds to me that this paper is at least a defect report in more generalized meaning.